app/controllers/subscriptions_controller.rb

```diff
@@ -86,7 +86,7 @@ def notify_new_subscription(subscription)
 
   def trigger_create_job_alert_clicked_event
     request_event.trigger(:vacancy_create_job_alert_clicked, vacancy_id: StringAnonymiser.new(vacancy_id))
-    trigger_dfe_analytics_event(:vacancy_create_job_alert_clicked, { vacancy_id: StringAnonymiser.new(vacancy_id) })
+    trigger_dfe_analytics_event(:vacancy_create_job_alert_clicked, { vacancy_id: StringAnonymiser.new(vacancy_id).to_s })
```

Comment:
Is the Vacancy ID a PII?  We are removing the anonymiser calls on it further down the PR.

Reply:
Agree with @scruti. It's just id so does not need to be anonymised